API.md

```diff
@@ -363,6 +363,17 @@ LoRa.enableInvertIQ();
 
 LoRa.disableInvertIQ();
 ```
+### LNA Gain
+
+Set LNA Gain for better RX sensitivity, by default ADC is used and LNA Gain is not used.
+
+```arduino
+LoRa.setGain(gain);
+```
+
+ * `gain` - LNA gain
+
+Supported values are between `0` and `6`. If gain is 0, ADC will be enabled and LNA gain will not be used. Else if gain is from 1 to 6, ADC will be disabled and LNA gain will be used.
```

Comment:
Based on your last comment is it `ADC` or `AGC` (automatic gain control) ?

Reply:
My bad. AGC Automatic Gain Control.
Let me fix the API wording

Reply:
Fixed. AGC is Automatic Gain Control. 
https://github.com/sandeepmistry/arduino-LoRa/pull/408/commits/97e83a33863d31702ab641d13408602f41dc5514